comptoir-ejb/src/main/java/be/valuya/comptoir/model/commercial/ItemVariantSale.java

```diff
@@ -48,6 +42,9 @@
     @NotNull
     @Nonnull
     private Stock stock;
+    @Column(name = "FORCE_CUSTOMER_LOYALTY")
+    @CheckForNull
+    private Boolean forceCustomerLoyalty;
```

Comment:
Should be a small cap boolean primitive, which can't be null.

Le jeu. 24 mars 2016 19:21, cghislai <notifications@github.com> a écrit :

> In
> comptoir-ejb/src/main/java/be/valuya/comptoir/model/commercial/ItemVariantSale.java
> <https://github.com/Valuya/comptoir/pull/139#discussion_r57363889>:
>
> > @@ -48,6 +42,9 @@
> >      @NotNull
> >      @Nonnull
> >      private Stock stock;
> > +    @Column(name = "FORCE_CUSTOMER_LOYALTY")
> > +    @CheckForNull
> > +    private Boolean forceCustomerLoyalty;
>
> Thinking of it, this should be a non-nullable 'useForCustomerLoyalty'
> field, set by the backend once the itemSale is created.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Valuya/comptoir/pull/139/files/fdef627be078769467ecb5f4a413dfb37f2e0a93#r57363889>
>
-- 
Yannick Majoros
Valuya sprl